lxd/storage/drivers/driver_cephfs.go

```diff
@@ -67,8 +67,8 @@ func (d *cephfs) Info() Info {
 	}
 }
 
-func (d *cephfs) HasVolume(volType VolumeType, volName string) bool {
-	if shared.PathExists(GetVolumeMountPath(d.name, volType, volName)) {
+func (d *cephfs) HasVolume(vol Volume) bool {
+	if shared.PathExists(GetVolumeMountPath(d.name, vol.volType, vol.name)) {
```

Comment:
@monstermunchkin just a nit, you could just use `vol.MountPath()` here rather than `GetVolumeMountPath`

Reply:
Sure, makes sense.